### PR TITLE
Switch from browserify to webpack

### PR DESCRIPTION
**Pull request by blink1073**

### CODE REVIEW

**Comment by blink1073** [comment]
Updated.

File: README.md [hunk 1]
```diff
@@ -8,6 +8,8 @@ A Phosphor layout panel which arranges its children into a single row or column.
 
 [API Docs](http://phosphorjs.github.io/phosphor-boxpanel/api/)
 
+[Live Example](http://phosphorjs.github.io/phosphor-boxpanel/example/)
```
Comment (sccolbert) [inline README.md]: Please remove this. Not sure we want to publish these examples yet.

File: scripts/travis_after_success.sh [hunk 2]
```diff
@@ -20,6 +20,7 @@ then
 
     rm -rf ./*
     cp -r ../docs/* ./.
+    cp -r ../example/ example/
```
Comment (sccolbert) [inline scripts/travis_after_success.sh]: Please remove this. Not sure we want to publish these examples yet.

File: scripts/travis_script.sh [hunk 3]
```diff
@@ -6,4 +6,6 @@ npm run clean
 npm run build
 npm test
 npm run test:coverage
+npm run clean:example
```
Comment (sccolbert) [inline scripts/travis_script.sh]: `clean` not needed

**Comment by blink1073** [comment]
All comments addressed.

File: scripts/travis_after_success.sh [hunk 4]
```diff
@@ -20,6 +20,7 @@ then
 
     rm -rf ./*
     cp -r ../docs/* ./.
+
```
Comment (sccolbert) [inline scripts/travis_after_success.sh]: Can you prune this file from the diff. There is no effective change.

**Pull request merged**